dropwizard-core/src/main/java/io/dropwizard/server/AbstractServerFactory.java

```diff
@@ -459,7 +459,14 @@ protected Handler createAppServlet(Server server,
             handler.addFilter(holder, "/*", EnumSet.allOf(DispatcherType.class));
         }
         if (jerseyContainer != null) {
-            jersey.setUrlPattern(jerseyRootPath);
+            String urlPattern = jerseyRootPath;
+            if (!urlPattern.endsWith("/")) {
```

Comment:
Unless I'm reading this wrong, if the user provide a rootPath  ending with * this will add another /, then it will add another *. We shouldn't add the slash if it already ends with *